docs/example-scenario/ai/newsfeed-ingestion.md

```diff
@@ -0,0 +1,161 @@
+
+### Scalability
+
+Function app scaling depends on the [hosting plan][plan] you use. This solution assumes a [Consumption plan][plan-c], in which compute power is automatically allocated to the functions when required. You pay only when your functions are running. Another option is to use an [Azure App Service][plan-aas] plan, which allows you to scale between tiers to allocate a different amount of resources.
+
+With Azure Cosmos DB, the key is to distribute your workload roughly evenly among a sufficiently large number of [partition keys][keys]. There's no limit to the total amount of data that a container can store or to the total amount of
+[throughput][throughput] that a container can support.
+
+### Management and logging
+
+This solution uses [Azure Application Insights][aai] to collect performance and logging information. An instance of Application Insights is created with the deployment in the same resource group as the other services needed for this deployment.
+
+To view the logs generated by the solution:
+
+1. Go to [Azure portal][portal] and navigate to the resource group created for the deployment.
+
+2. Click the **Application Insights** instance.
+
+3. From the **Application Insights** section, navigate to **Investigate\\Search** and search the data.
+
+### Security
+
+Azure Cosmos DB uses a secured connection and key through the C\# SDK provided by Microsoft. There are no other externally facing surface areas. Learn more about security [best practices][db-practices] for Azure Cosmos DB.
+
+## Pricing
+
+The estimated daily cost to keep this deployment available is approximately \$20 U.S. with no data moving through the system.
+
+Azure Cosmos DB is incredibly powerful but incurs the greatest [cost][db-cost] in this deployment. You can use another storage solution by refactoring the Azure Function code provided.
+
+Pricing for Azure Functions varies depending on the [plan][function-plan] it runs in.
+
+## Deploy the scenario
+
+> [!NOTE]
+> You must have an existing Azure account. If you don't have an Azure subscription, create a [free account][free] before you begin.
+
+All the code for this scenario is available in the [GitHub][github]. This repository contains the source code used to build the generator application that feeds the pipeline for this demo.
+
+### Azure Cosmos DB considerations
+
+The GitHub [readme][github] discusses several required data formats.
+
+The two core collections for the articles pipeline are Ingest and Processed. The design of the two are similar in that there are few first-class members of the document but potentially many *properties* associated with the document depending on the type.
```

Comment:
This is the first time "collections" are being introduced in the context of the pipeline. This concept needs to be introduced and fleshed out more. The discussion of member types in the document is confusing because they haven't yet been introduced.

Reply:
That's probably my fault for not pointing that out. Yes, Cosmos is set up using SQL API so this is a DB collection as described here:

https://docs.microsoft.com/en-us/azure/cosmos-db/databases-containers-items

Hopefully that article helps.

Reply:
This feels more like implementation specific guidance that should be in Github.  Removing from the article.